package.json

```diff
@@ -1,63 +1,49 @@
 {
 	"name": "@monogram/eslint-config",
-	"version": "0.1.2",
+	"version": "0.2.0",
 	"description": "ESLint configs used at Monogram",
 	"keywords": [
 		"eslint",
-		"eslint-config-airbnb",
-		"eslint-config-next"
+		"eslintconfig",
+		"eslint-config"
 	],
 	"license": "MIT",
 	"scripts": {
-		"update-local": "pnpm build && pnpm install-locally",
-		"install-locally": "npm install -g",
-		"prepublish": "pnpm build && pnpm release",
-		"build": "tsc --project tsconfig.json",
-		"prestart": "pnpm build",
-		"start": "ts-node src/index.ts",
-		"release": "standard-version",
-		"postinstall": "pnpm build"
-	},
-	"main": "dist/index.js",
-	"bin": {
-		"eslint-config": "dist/index.js"
-	},
-	"files": [
-		"dist"
-	],
-	"peerDependencies": {
-		"@typescript-eslint/eslint-plugin": "^5.30.7",
-		"@typescript-eslint/parser": "^5.30.7",
-		"eslint": "^8.20.0",
-		"eslint-config-airbnb-base": "^15.0.0",
-		"eslint-config-prettier": "^8.5.0",
-		"eslint-import-resolver-typescript": "^3.3.0",
-		"eslint-plugin-import": "^2.26.0"
+		"install-locally": "pnpm build && npm install -g",
+		"prepublish": "pnpm build",
+		"build": "tsc"
 	},
+	"main": "build/index.js",
+	"bin": "build/index.js",
 	"devDependencies": {
-		"@commitlint/cli": "^17.0.3",
-		"@commitlint/config-conventional": "^17.0.3",
-		"@commitlint/types": "^16.2.1",
-		"@monogram/prettier-config": "^0.0.8",
-		"@types/node": "^17.0.18",
-		"@typescript-eslint/eslint-plugin": "^5.30.7",
-		"@typescript-eslint/parser": "^5.30.7",
-		"eslint": "^8.20.0",
+		"@commitlint/cli": "^17.6.7",
+		"@commitlint/config-conventional": "^17.6.7",
+		"@commitlint/types": "^17.4.4",
+		"@monogram/prettier-config": "^0.2.0",
+		"@types/node": "^20.4.5",
+		"@types/prompts": "^2.4.4",
+		"typescript": "^5.1.6"
+	},
+	"dependencies": {
+		"@typescript-eslint/eslint-plugin": "^6.1.0",
+		"@typescript-eslint/parser": "6.1.0",
+		"eslint-config-airbnb": "^19.0.4",
 		"eslint-config-airbnb-base": "^15.0.0",
-		"eslint-config-prettier": "^8.5.0",
-		"eslint-import-resolver-typescript": "^3.3.0",
-		"eslint-plugin-import": "^2.26.0",
-		"husky": "^7.0.4",
-		"standard-version": "^9.3.2",
-		"ts-node": "^10.5.0",
-		"typescript": "^4.5.5"
+		"eslint-config-next": "^13.4.11",
+		"eslint-config-prettier": "^8.8.0",
+		"eslint-import-resolver-typescript": "^3.5.5",
+		"eslint-plugin-import": "^2.27.5",
+		"prompts": "^2.4.2"
```

Comment:
looks like [Inquirer.js](https://github.com/SBoudrias/Inquirer.js) is a better option at the moment as it's actively maintained, can we switch to that without too much work?

Reply:
Yeah good observation, I'll switch it over, should be trivial